src/components/Unauthenticated.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { useFusionAuth } from '../providers/FusionAuthProvider';
+
+export const Unauthenticated: FC<PropsWithChildren> = props => {
+    const { isAuthenticated } = useFusionAuth();
+
+    if (isAuthenticated) {
+        return null;
+    }
+
+    return <>{props.children}</>;
```

Comment:
@JakeLo123 Let's boil this down to a single `return` statement just to simply for maintainers and cut down on possible code execution paths.

```tsx
export const Unauthenticated: FC<PropsWithChildren> = props => {
    const { isAuthenticated } = useFusionAuth();

    return isAuthenticated ? null : <>{props.children}</>;
};
```